CHANGELOG.md

```diff
@@ -9,9 +9,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Breaking Changes
```

Comment:
test that older backend SDK tests still work with the new core

Reply:
Ran all tests with the old node SDK (master). All tests passed, the only change required was the name of core config values we set in the tests.